shared_tasks/dynalab/model_generation.json

```diff
@@ -1,7 +1,25 @@
 {
     "dummy": false,
-    "beam_size": 1,
-    "max_len_a": 1.3,
-    "max_len_b": 5,
-    "min_len": 5
-}
+    "generation": {
+        "beam_size": 1,
+        "max_len_a": 3,
+        "max_len_b": 5,
+        "min_len": 1,
+        "unk_penalty": 100.0
+    },
+    "directions":
```

Comment:
those are directions for Wikipedia translation not for the african task. I don't think we need this filtering here. Please remove, here and in handler.py